src/notebooks/readme.md

```diff
@@ -0,0 +1,25 @@
+# notebooks
+
+These notebooks provide examples of how to work with the CSV files that are
+generated by the extractor programs.
+
+1. Ensure you have [Python 3.8+ and Poetry](../README.md#getting-started)
+1. At a command prompt, install all required dependencies:

```

Comment:
Same comment about adding the directory where the commands should be executed from.

Reply:
done